caffe2/python/operator_test/conv_test.py

```diff
@@ -118,14 +141,20 @@ def test_convolution_separate_stride_pad_gradients(self, op_type,
            batch_size=st.integers(1, 3),
            engine=st.sampled_from(["", "EIGEN"]),
            use_bias=st.booleans(),
+           force_algo_fwd=_cudnn_convolution_algo_count("fwd"),
+           force_algo_dgrad=_cudnn_convolution_algo_count("dgrad"),
+           force_algo_wgrad=_cudnn_convolution_algo_count("wgrad"),
            **hu.gcs)
     def test_convolution_separate_stride_pad_layout(self, op_type,
```

Comment:
same as `test_convolution_separate_stride_pad_gradients`, not test with "CUDNN" engine

Reply:
Ok.